docs/source/project_documentation/home_page_index.rst

```diff
@@ -0,0 +1,158 @@
+
+We know we are successful when we see:
+
+* rigorous configuration management of MBSE artifacts
+* reduced MBSE work / rework
+* upstream MBSE artifacts actively used by product development & delivery teams
+* implementation of automated quality assurance in pipelines using MBSE artifacts
+* and frequent delivery of complete system engineering, implementation, and test baselines with full traceability.
+
+We believe we can discover new ways to define, deliver, and evolve complex systems using Architecture-as-Code.
+
+The Command Line Interface
+--------------------------
+
+AaC is also the name of the modular command line interface (CLI) that can parse a Architecture-as-Code model
+and produce artifacts from it. The CLI has an extensible command library that currently contains commands that
+can verify syntax and patterns, generate code and produce static reports.
+
+The entire AaC product is extensible.  AaC is purposefully small.  Additional functionality is achieved through installing plugins.
+The plugin architecture allows you choice of functionality so you can use only the features you want.
+
+The Graphical User Interface
+----------------------------
+
+AaC has a backlog item to create a GUI, but it has not yet been built.  We are currently performing interviews
+with stakeholders to determine the best way to provide them value quickly.  Stay tuned for new releases.
+
+AaC is Extensible
+-----------------
+
+AaC is designed with extensibility in mind.  The built-in functionality is intentionally minimized.
+AaC uses a plug-in system to extend the base capability.  To further simplify this, AaC includes a
+built-in command to generate new plugins from an AaC plugin model.  Our own built-in plugins are developed
+using this methodology, so you can see examples in the GitHub repo.
+
+We have intentionally reduced the number of built-in plugins and transitioned them to separate repositories.
+We want users to have more control over what AaC functionallity they utilize (i.e. opt-in) rather than
+put a bunch of stuff in the default installation that you may not want or need.  As we evolve AaC, you can look for
+our extensions in the `DevOps-MBSE GitHub Organization <https://github.com/orgs/DevOps-MBSE/repositories>`_.
+
+About the project
+=================
+
+.. include:: <isonum.txt>
+
+Architecture-as-Code (AaC) is |copy| 2021-present by AaC Project Contributors.
+
+License
+-------
+
+Arcitecture-as-Code is distributed by an `MIT license <https://github.com/DevOps-MBSE/AaC/blob/main/LICENSE>`_.
+
+Contributing
+------------
+
+When contributing to this repository, please first discuss the change you wish to make via issue or discussion topic with the owners of this repository before making a change. Read more about becoming a contributor on `GitHub contributions <https://github.com/pmarsceill/just-the-docs#contributing>`_ and see our `current contribution status <https://github.com/jondavid-black/AaC#note-to-contributors>`_.

```

Comment:
Need to update that last link to be to the rest-api repo contributor note that we added to be inherited from the org level.